src/core/qgsbrowsermodel.h

```diff
@@ -162,6 +164,15 @@ class CORE_EXPORT QgsBrowserModel : public QAbstractItemModel
     //! Reload the whole model
     void reload();
 
+    /**
+     * Add a new provider to the model.
```

Comment:
@param /@return are bad :-1: Just go:

    * Adds a new \a provider to the model
    * \since QGIS 3.4

Also dox for removeProvider